metricsproject/Untitled.ipynb

```diff
@@ -0,0 +1,175 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "cm = confusion_matrix.unit_test(y_true=y_test.tolist(), y_pred=y_pred.tolist(), title='Confusion matrix, without normalization', normalize=False, classes=class_names.tolist())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 57,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'matrix': [[0.0, 1.0, 2.0], [0.0, 1.0, 2.0], [0.0, 1.0, 2.0]], 'visual': metadata {\n",
+      "  type {\n",
+      "  }\n",
+      "}\n",
+      "uri: \"/var/folders/2b/wytvyh952jz8c17mp6yvh8nc0000gq/T/test_8wu_53cn/visual\"\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(cm)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "FileNotFoundError",
+     "evalue": "[Errno 2] No such file or directory: '/var/folders/2b/wytvyh952jz8c17mp6yvh8nc0000gq/T/test_inbo8f8i/visual'",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
```

Comment:
Is showing an error here intended?

Reply:
ohh i did not want to commit the notebook